doc/services/portability/posix/overview/index.rst

```diff
@@ -0,0 +1,134 @@
+.. _posix_overview:
+
+Overview
+########
+
+The Portable Operating System Interface (POSIX) is a family of standards specified by the
+`IEEE Computer Society`_ for maintaining compatibility between operating systems. Zephyr
+implements a subset of the standard POSIX API specified by `IEEE 1003.1-2017`_ (also known as
+POSIX-1.2017).
+
+..  figure:: posix.svg
+    :align: center
+    :alt: POSIX Support in Zephyr
+
+    POSIX support in Zephyr
+
+.. note::
+    This page does not document Zephyr's :ref:`POSIX architecture<Posix arch>`, which is used to
+    run Zephyr as a native application under the host operating system for prototyping,
+    test, and diagnostic purposes.
+
+With the POSIX support available in Zephyr, an existing POSIX conformant
+application can be ported to run on the Zephyr kernel, and therefore leverage
+Zephyr features and functionality. Additionally, a library designed to be
+POSIX conformant can be ported to Zephyr kernel based applications with no changes.
+
+The POSIX API is an increasingly popular OSAL (operating system abstraction layer) for IoT and
+embedded applications, as can be seen in Zephyr, AWS:FreeRTOS, TI-RTOS, and NuttX.
+
+Benefits of POSIX support in Zephyr include:
+
+- Offering a familiar API to non-embedded programmers, especially from Linux
+- Enabling reuse (portability) of existing libraries based on POSIX APIs
+- Providing an efficient API subset appropriate for small (MCU) embedded systems
+
+.. _posix_subprofiles:
+
+POSIX Subprofiles
+=================
+
+While Zephyr supports running multiple `threads <threads_v2>`_ (possibly in an `SMP <smp_arch>`_
+configuration), as well as `Virtual Memory and MMUs<memory_management_api>`_, Zephyr code and data
```

Comment:
this breaks the doc build

```suggestion
configuration), as well as `Virtual Memory and MMUs <memory_management_api>`_, Zephyr code and data
```

Reply:
Oh weird - thanks for noticing. I was able to build fine locally.

Reply:
Ya... feels like this might be a parsing bug/regression in Sphinx or docutils tbh

Reply:
Fixed!